### ISSUE TITLE
Have C-c C-a run asynchronous

### ISSUE THREAD

**Issue by dinojr** [Malabarba/latex-extra#25]
I noticed that compile-commands-until-done sets TeX-process-asynchronous to nil. When I hack it to set it to true, I sometimes get some concurrent LaTeX process running at the same time. Could it be avoided ? It's a pity having to wait for a long compile time when one could be writing the next part.

**Comment by Malabarba** [comment]
Yes, that can be done by rewriting the function in terms of a process sentinel (or maybe hooks). It would take a bit of work though, and I probably won't have time for it. 

PRs welcome.

**Comment by dinojr** [comment]
Artur Malabarba <notifications@github.com> writes:

> Yes, that can be done by rewriting the function in terms of a process
> sentinel (or maybe hooks). It would take a bit of work though, and I
> probably won't have time for it.

Maybe I'll give it a try. Never used a sentinel before though.

Julien.
